### ISSUE TITLE
Error after SwiftBackup

### ISSUE THREAD

**Issue by TanjaZ84** [freeotp/freeotp-android#411]
I have a Samsung with root access that I updated for which i formatted data. Ich backed everything up with Swift. The App restored and I can see the accounts but when I push on an account to get a OTP I get ERROR. I have seen a identical description of this error in closed issues where the solution was to use an older backup because it seemed the backup was compromised. I have tried that but did not get a better result. (The backups of other apps worked fine.)

**Comment by vdbhb59** [comment]
2 things, I have seen (work in FreeOTP+):
1. Remove the authentication option in the app, and then backup and restore it. It will work. The encryption seems to be broken when done with authentication ON, especially with fingerprint authentication.
2. After backup see if the entries are correct (open via any text editor) and see. Correct the entries and save and restore. Will work.

**Comment by BeeGrech** [comment]
@vdbhb59 -- freeotp doesnt have auth and backup

**Comment by vdbhb59** [comment]
Yeah, I kind of confused between the 2, apologies on that part. Ignore that point, but the backup does have issues of malformed.

**Comment by halfsquatch** [comment]
I cannot backup and restore. I guess I will have to use Google Authenticator again. Oh well.

**Comment by steve-contrib** [comment]
You can solve the freeOTP import problem by importing your "externalBackup.xml" file from https://github.com/stratumauth/app
